src/autocomplete/StreamAutocomplete.js

```diff
@@ -14,11 +14,13 @@ class StreamAutocomplete extends Component {
   render() {
     const { filter, subscriptions, onAutocomplete } = this.props;
     const streams = subscriptions
-      .filter(x => x.name.toLowerCase().startsWith(filter.toLowerCase()))
-      .slice(0, 5);
```

Comment:
We can use `ListView` instead of `map` which have better performance. (tested on both Android as well as iOS)